src/components/publishing/sections/image_wrapper.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import styled from "styled-components"
+import ViewFullscreen from "./view_fullscreen"
+
+interface ImageWrapperProps extends React.HTMLProps<HTMLDivElement> {
+  src: any
+  layout?: string
```

Comment:
Maybe type this as a union of the possible strings so it’s documented when people try to use it.

Reply:
I see that `layout` is used everywhere, so in case you do decide to define a union, maybe it’s worth it to create a central typings file for this which you can import wherever this typing is used.